server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -793,7 +793,7 @@ callUsingPhrase
    ;
 
 callUsingParameter
-   : callByReferencePhrase | callByValuePhrase | callByContentPhrase
+   : (callByReferencePhrase | callByValuePhrase | callByContentPhrase) COMMACHAR?
```

Comment:
The change will fix the error, but this grammar matches a lot of cases:
```
VAR1, VAR2
VAR1, VAR2,
VAR1 VAR2
```
Probably, it is not correct.

Reply:
I would expect it to be:
```
callUsingPhrase
   : USING callUsingParameter (COMMACHAR? callUsingParameter)*
   ;
```
Should be checked that the positive tests pass and the loose comma at the end produces a syntax error